### PR TITLE
Adds example usage and tests for v1beta2

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: language/analysis/src/main/java/com/google/cloud/language/samples/AnalyzeBeta.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (jabubake) [inline language/analysis/src/main/java/com/google/cloud/language/samples/AnalyzeBeta.java]: We're following the convention of not having  "All Rights Reserved"  license in java-docs-samples

File: language/analysis/demo-beta.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Demonstrates how to run the AnalyzeBeta sample.
+
+##########################################################################
+# Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (jabubake) [inline language/analysis/demo-beta.sh]: Please remove "All Rights Reserved."

**Comment by gguuss** [comment]
@jabubake thank you for the review
@lesv FYI, examples for NL beta

**Comment by lesv** [comment]
LGTM

**Pull request merged**